src/view/configuration/configuration.jsx

```diff
@@ -528,7 +537,23 @@ const Configuration = () => {
                           </div> */}
 
                           <h3>Context</h3>
-
+                          <div className="u-gapTop">
+                            <InfoTipLayout tip="If you want to add, remove, or modify fields from the event globally, you can configure an onBeforeEventSend callback.  This callback will be called everytime and event is sent.  This callback passes an object with an `xdm` fields.  Modify the xdm object to change the data that is sent in the event.">
+                              <FieldLabel
+                                labelFor="onBeforeEventSendField"
+                                label="Set this to configure a callback that will be called for every event."
```

Comment:
It's a bit odd for a label to be telling the customer what to do. It should probably describe the feature. This is a tricky one though. Maybe something like "Callback function for modifying data before each event is sent to the server" (no period to be consistent with the other labels)